src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task{
+
+    protected String from;
+    protected String to;
```

Comment:
from and to can be confusing to read when the toString() returns " (from: " + from + " to: " + to + ")";